src/org/company/MyClass.java

```diff
@@ -1,5 +1,7 @@
 package org.company;
 
+import com.sun.istack.internal.NotNull;
```

Comment:
I don't know. A method below is using them and they are not imported. I used IntelliJ to auto-import them to resolve compiler problems.